index.js

```diff
@@ -1,19 +1,39 @@
+const yaml = require('js-yaml');
+
 module.exports = robot => {
     robot.on('pull_request.opened', receive);
     robot.on('issues.opened', receive);
     async function receive(context) {
-        const title = context.payload.issue.title;
-        const body = context.payload.issue.body;
-        // robot.log('Title: ', title, 'Body: ', body);
-
-        // Add options for if there is a default title?
+        let title, body, number, bodies, badTitle;
+        if (context.payload.pull_request) {
+            ({ title, body, number } = context.payload.pull_request);
+        } else {
+            ({ title, body, number } = context.payload.issue);
+        }
 
-        if (!body) {
-            const options = context.repo({path: '.github/request-info.md'});
+        try {
+            const options = context.repo({path: '.github/config.yml'});
             const response = await context.github.repos.getContent(options);
-            const template = Buffer.from(response.data.content, 'base64').toString();
+            bodies = yaml.load(Buffer.from(response.data.content, 'base64').toString()) || {};
+        } catch (err) {
+            if (err.code !== 404) throw err;
+        }
 
-            context.github.issues.createComment(context.issue({body: template}));
+        if (bodies) {
```

Comment:
It's a little confusing to have something related to `bodies` here and then `body` in the lines below. Maybe breaking these into their own functions might help with readability?